### ISSUE TITLE
The instance segmentation model reported in the paper?

### ISSUE THREAD

**Issue by jcdubron** [bdd100k/bdd100k#160]
It's reported in Table 6 of your paper that using the 7k BDD instance segmentation dataset could reach AP = 21.8. However, I cannot reproduce this under the default configuration of Mask R-CNN in Detectron2. Could you provide more implementation details? Thanks.

**Comment by XiaLiPKU** [comment]
Hi, the main differences arise from the changes in evaluation.

In the paper, we adopt the coco-format ground truth and use COCO scripts for evaluation.
However, since [PR 36](https://github.com/bdd100k/bdd100k/pull/36), we introduce the usage of [Bitmask](https://doc.bdd100k.com/format.html#instance-segmentation). Now, the evaluation runs on predictions and ground truth with the Bitmask format.

These two formats have obvious differences. For the coco format, the predictions of different instances can overlap. However, using our bitmask format, one pixel only belongs to one instance.

Currently, we have mmdet-pretrained MaskRCNNs to be published. MaskRCNN with ResNet50, trained with a 1x schedule, can achieve 16.8 in AP.

**Comment by jcdubron** [comment]
Thanks for your detailed explanation. 

I adopt the coco-format ground truth for training and evaluation, which is converted from bitmask format by the official script. The Mask R-CNN with ResNet50 and 1x schedule reaches AP = 19.8. It would be great if your training code could be published.

**Comment by XiaLiPKU** [comment]
The coco-format json from polygon is different from 'the one from bitmask'.
Polygons allow for overlaps, but Bitmask doesn't. So that's source of
difference.

DuBron ***@***.***> 于2021年8月24日周二 上午11:52写道：

**Comment by jcdubron** [comment]
I see. Is there any script to generate coco format from polygons? Or did you convert the format with something like `poly2patch` function in [link](https://github.com/bdd100k/bdd100k/pull/36/files#diff-70f6ac12fcbdd0b52412aaba9e770edb9044463ea1092ffea11ef4ec8b845b7b) and then to rle with `cocoapi`?

**Comment by XiaLiPKU** [comment]
Yes. In scalabel, we provide this one: https://doc.scalabel.ai/label.html#to-coco.

However, we don't advise our users to do so. Because our official evaluation adopts the bitmask format.

**Comment by jcdubron** [comment]
Thanks a lot.
